src/pages/Settings.jsx

```diff
@@ -7,12 +11,56 @@ import {
 } from "@/constants/settingOptions";
 
 const Settings = () => {
+  const [selectedRedirectChannelId, setSelectedRedirectChannelId] =
+    useState(null);
   const settingTypes = Object.values(SETTING_TYPES);
 
+  const handleSelectRedirectChannel = (channelId) => {
+    if (channelId === selectedRedirectChannelId) {
+      setSelectedRedirectChannelId(null);
+      return;
+    }
+
+    setSelectedRedirectChannelId(channelId);
+    // TODO: 동적 구현 시 update api 연결
+  };
```

Comment:
```suggestion
  setSelectedRedirectChannelId((prevId) =>
    prevId === channelId ? null : channelId
  );
```

선택된 채널을 토글하는 역할이라면 함수형 업데이트를 적용해도 좋을 것 같습니다!
삼항 연산자를 사용하면 조건 분기 처리가 간결해지고 가독성 측면에서도 더 명확해질 것 같아요😊

Reply:
좋은 의견 감사합니다!
말씀해주신 대로 함수형 업데이트와 삼항 연산자 적용으로 코드가 더 간결하고 읽기 쉬워진 것 같아요.
반영해서 수정했습니다!